lib/discourse_slack/slack.rb

```diff
@@ -40,7 +40,7 @@ def self.status
       end
 
       get_store.each do |row|
-        tag_list = row['tags'].present? ? row['tags'].join(', ') : ""
+        tag_list = row['tags'].present? ? "with tags *" + row['tags'].join(', ') + "*" : ""
```

Comment:
We've duplicated logic in two places. While we're at it, can we see if we can refactor the code so that we don't duplicate logic?